recipes/libpq/all/test_package/test_package.c

```diff
@@ -13,4 +14,4 @@ int main() {
     
     PQfinish(conn);
     return EXIT_SUCCESS;
-}
\ No newline at end of file
+}
```

Comment:
for @uilianries 
probably, in future we may add hook check for `no new line at the end of file`.
this sometimes emits compiler warnings, and may fail for certain users (e.g. whose compile with **-Werror**).

Reply:
Yes, please!

Reply:
Maybe pylint is able to do it, so we could active that hook for conan-center.

Reply:
Now, I see, do you mean for C and C++ files. Yes, I agree.

Reply:
I would include all common text files. *.py, *.c, *.cpp, *.cc, *.cxx, *.yml, ..
Linters probably have some option for this.
If not, I'ld limit to these file type.

Reply:
I've opened conan-io/hooks#161